Bots/DotNet/Skills/CodeFirst/TeamsWaterfallSkillBot/TeamsWaterfallSkillBot.csproj

```diff
@@ -0,0 +1,47 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <AssemblyName>Microsoft.BotFrameworkFunctionalTests.WaterfallSkillBot</AssemblyName>
```

Comment:
The assemble and namespace don't match the project name, Microsoft.BotFrameworkFunctionalTests.TeamsWaterfallSkillBot

Reply:
updated